CryptoPkg/CryptoPkg.ci.yaml

```diff
@@ -68,15 +75,68 @@
     },
     "DscCompleteCheck": {
         "DscPath": "CryptoPkg.dsc",
-        "IgnoreInf": []
+        "IgnoreInf": [
+            "CryptoPkg/Library/BaseCryptLibMbedTls/BaseCryptLib.inf",
+            "CryptoPkg/Library/BaseCryptLibMbedTls/PeiCryptLib.inf",
+            "CryptoPkg/Library/BaseCryptLibMbedTls/RuntimeCryptLib.inf",
+            "CryptoPkg/Library/BaseCryptLibMbedTls/SecCryptLib.inf",
+            "CryptoPkg/Library/BaseCryptLibMbedTls/SmmCryptLib.inf",
+            "CryptoPkg/Library/BaseCryptLibMbedTls/TestBaseCryptLib.inf",
+            "CryptoPkg/Library/MbedTlsLib/MbedTlsLib.inf",
+            "CryptoPkg/Library/MbedTlsLib/MbedTlsLibFull.inf"
+        ]
     },
     "GuidCheck": {
         "IgnoreGuidName": [],
         "IgnoreGuidValue": [],
         "IgnoreFoldersAndFiles": []
     },
     "LibraryClassCheck": {
-        "IgnoreHeaderFile": []
+        "IgnoreHeaderFile": [

```

Comment:
I believe you are getting a similar CI error with the INF, the solution is same as above

Reply:
Thanks. I have fixed this issue.